shared/templates/audit_rules_syscall_events/bash.template

```diff
@@ -14,6 +11,6 @@ do
 	FULL_RULE="-a always,exit -F arch=$ARCH -S {{{ ATTR }}} -F auid>={{{ auid }}} -F auid!=4294967295 -F key=perm_mod"
 
 	# Perform the remediation for both possible tools: 'auditctl' and 'augenrules'
-	fix_audit_syscall_rule "augenrules" "$PATTERN" "$GROUP" "$ARCH" "$FULL_RULE"
-	fix_audit_syscall_rule "auditctl" "$PATTERN" "$GROUP" "$ARCH" "$FULL_RULE"
+	{{{ bash_fix_audit_syscall_rule("augenrules","$PATTERN","$GROUP","$ARCH","$FULL_RULE") }}}
+	{{{ bash_fix_audit_syscall_rule("auditctl","$PATTERN","$GROUP","$ARCH","$FULL_RULE") }}}
```

Comment:
I noticed that the macro `bash_fix_audit_syscall_rule` has 7 parameters now but you only pass 5 parameters. That doesn't seem right. See it's definition:
 https://github.com/ComplianceAsCode/content/blob/87d83f35ba7fa5f9545e7ab8472ce922851e8348/shared/macros/10-bash.jinja#L1475

Reply:
The bad include is found e.g. in bash/sle15-script-stig.sh  (nowhere else i see it via grep)